app/controllers/api/public/v1/providers/courses_controller.rb

```diff
@@ -39,6 +40,10 @@ def provider
           def recruitment_cycle
             @recruitment_cycle ||= RecruitmentCycle.find_by(year: params[:recruitment_cycle_year])
           end
+
+          def include_param
```

Comment:
do we need to do something like 

https://github.com/DFE-Digital/teacher-training-api/blob/5748d7c3b8e9de80e701bf3f869899d8cd818833/app/controllers/api/public/v1/providers/locations_controller.rb#L33-L37

Reply:
I don't see why this is needed. The serialiser already specifies which associations can be serialised. Adding `PERMITTED_INCLUSIONS` would simply do the same work twice